hydradx/model/indexer_utils.py

```diff
@@ -40,12 +46,14 @@ def query_indexer(url: str, query: str, variables: dict = None) -> dict:
     return return_val
 
 
-def get_asset_info_by_ids(asset_ids: list) -> dict:
+def get_asset_info_by_ids(asset_ids: list = None) -> dict:
```

Comment:
maybe add really basic test for this? Like, check that doing this query gives you a non-empty list of assets or something simple

Reply:
done